.gitignore

```diff
@@ -19,7 +19,6 @@ test/coverage
 data
 doc/source/modules.rst
 doc/source/rho.rst
-doc/facts.rst
```

Comment:
Can we please merge the changes from the most recent PR to avoid removing this line?

Reply:
I assumed you missed this in your last PR as all the facts are now missing in the command syntax page where you had included them when you go to view this on Github.

Reply:
Sorry I wasn't clear about this. I had deliberately left `doc/facts.rst` out of the repository because the information is there in `rho/facts.py`, and the Makefile can generate it on demand if we need the reST version. I'm happy to have it checked-in if you'd rather do that.

Reply:
Yes, I'd prefer to have it as I think there is more value in having the facts list available in the Github documentation.